src/test/resources/projects/multi-mc-versions/settings.gradle

```
@@ -1,3 +1,14 @@
+plugins {
+	id 'fabric-loom' apply false
+}
+
+gradle.lifecycle.beforeProject {
+	if (it.rootProject == it) return
+
+		apply plugin: 'fabric-loom'
```

```suggestion
	apply plugin: 'fabric-loom'
```
(to match the next line)